[DEFECT] Documentation is failing to build on MacOS

--------
Defect Description
--------
When running make_docs.sh in the HERON/doc directory, the documentation fails to build and instead yields the following stack trace.

##### What did you expect to see happen?
The documentation should have built and yielded a pdf in an adjacent directory.

##### What did you see instead?
```
Building manuals ...
... building in user_manual...
Traceback (most recent call last):
  File "script/generate_user_manual.py", line 9, in <module>
    import collect_tex
  File "/Users/grifaa/Documents/projects/HERON/doc/user_manual/script/collect_tex.py", line 15, in <module>
    import Cases, Components, Placeholders
  File "/Users/grifaa/Documents/projects/HERON/src/Cases.py", line 15, in <module>
    from HERON.src.base import Base
ModuleNotFoundError: No module named 'HERON'
make: *** [populate] Error 1
... ... failed to make docs in user_manual
```

##### Do you have a suggested fix for the development team?
Fix the import in the generate_user_manual.py file.

**Describe how to Reproduce**
Steps to reproduce the behavior:
1.  Change to directory HERON/doc
2.  run make_docs.sh

**Screenshots and Input Files**
Please attach the input file(s) that generate this error.  The simpler the input, the faster we can find the issue.

**Platform (please complete the following information):**
 - OS: macOS
 - Version: 12.4
 - Dependencies Installation: CONDA

----------------
For Change Control Board: Issue Review
----------------
This review should occur before any development is performed as a response to this issue.
- [ x] 1. Is it tagged with a type: defect or task?
- [ x] 2. Is it tagged with a priority: critical, normal or minor?
- [ x] 3. If it will impact requirements or requirements tests, is it tagged with requirements?
- [ x] 4. If it is a defect, can it cause wrong results for users? If so an email needs to be sent to the users.
- [ x] 5. Is a rationale provided? (Such as explaining why the improvement is needed or why current code is wrong.)

-------
For Change Control Board: Issue Closure
-------
This review should occur when the issue is imminently going to be closed.
- [x] 1. If the issue is a defect, is the defect fixed?
- [x] 2. If the issue is a defect, is the defect tested for in the regression test system?  (If not explain why not.)
- [x] 3. If the issue can impact users, has an email to the users group been written (the email should specify if the defect impacts stable or master)?
- [x] 4. If the issue is a defect, does it impact the latest release branch? If yes, is there any issue tagged with release (create if needed)?
- [x] 5. If the issue is being closed without a pull request, has an explanation of why it is being closed been provided?

--- a/doc/user_manual/script/collect_tex.py
+++ b/doc/user_manual/script/collect_tex.py
@@ -7,12 +7,12 @@
 import os
 import sys
 
-heron_path = os.path.abspath(os.path.join(os.path.dirname(os.path.abspath(__file__)), '..', '..', '..', 'src'))
+heron_path = os.path.abspath(os.path.join(os.path.dirname(os.path.abspath(__file__)), '..', '..', '..', '..'))
 build_path = os.path.abspath(os.path.join(os.path.dirname(os.path.abspath(__file__)), '..', 'build'))
 src_path = os.path.abspath(os.path.join(build_path, '..', 'src'))
 
 sys.path.append(heron_path)
-import Cases, Components, Placeholders
+from HERON.src import Cases, Components, Placeholders
 # import _utils as hutils
 # framework_path = hutils.get_raven_loc()
 # sys.path.append(framework_path)

--- a/doc/user_manual/script/collect_tex.py
+++ b/doc/user_manual/script/collect_tex.py
@@ -13,10 +13,6 @@ src_path = os.path.abspath(os.path.join(build_path, '..', 'src'))
 
 sys.path.append(heron_path)
 from HERON.src import Cases, Components, Placeholders
-# import _utils as hutils
-# framework_path = hutils.get_raven_loc()
-# sys.path.append(framework_path)
-# from utils import InputData
 
 specs_to_load = {'Cases': ['Case'],
                  'Components': ['Component'],

